tutorials/install-distro.md

```diff
@@ -64,11 +64,16 @@ To install on the following operating systems, set the environment variable $OS
 
 And then run the following as root:
 ```shell
-echo "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/ /" > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
-echo "deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable:/cri-o:/$VERSION/$OS/ /" > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable:cri-o:$VERSION.list
+sudo apt update -y
+sudo apt install ca-certificates -y
```

Comment:
Above we have the line "Note: this tutorial assumes you have curl and gnupg installed" 
I think we should drop that line, add a comment above this saying "these are prerequisites for installing" and then add those packages on this line as well